x-pack/packages/kbn-elastic-assistant-common/impl/schemas/knowledge_base/find_knowledge_base_entries_route.gen.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+import { ArrayFromString } from '@kbn/zod-helpers';
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ *
+ * info:
+ *   title: Find Knowledge Base Entries API endpoint
+ *   version: 1
+ */
+
+import { SortOrder } from '../common_attributes.gen';
+import { KnowledgeBaseEntryResponse } from './common_attributes.gen';
+
+export type FindKnowledgeBaseEntriesSortField = z.infer<typeof FindKnowledgeBaseEntriesSortField>;
+export const FindKnowledgeBaseEntriesSortField = z.enum([
+  'created_at',
+  'is_default',
+  'title',
+  'updated_at',
+]);
+export type FindKnowledgeBaseEntriesSortFieldEnum = typeof FindKnowledgeBaseEntriesSortField.enum;
+export const FindKnowledgeBaseEntriesSortFieldEnum = FindKnowledgeBaseEntriesSortField.enum;
+
+export type FindKnowledgeBaseEntriesRequestQuery = z.infer<
+  typeof FindKnowledgeBaseEntriesRequestQuery
+>;
+export const FindKnowledgeBaseEntriesRequestQuery = z.object({
```

Comment:
I would like to support semantic search in this API as well (already plumbed it out in kbDataClient over in the [sec-labs-kb PR](https://github.com/elastic/kibana/pull/184885/files#diff-655783627678b7bb8e827cafe0a930f448c42e264fc19bd8d796f8383537f4c3R19-R31)). Will probably expose something like `search_type:semantic` param to use it, and then re-use `filter` param. If there is a more standard pattern for supporting multiple search types in the same API I can use that. 

Not sure how pagination and stable references work with semantic search, so should verify that behavior first.